src/ui/common/src/components/LogViewer/index.tsx

```diff
@@ -7,25 +7,16 @@ import { Tab, Tabs } from '../primitives/Tabs.styles';
 type Props = {
   logs?: Logs;
   err?: Error;
-  contentHeight?: string;
 };
-const LogViewer: React.FC<Props> = ({ logs, err, contentHeight = '10vh' }) => {
+const LogViewer: React.FC<Props> = ({ logs, err }) => {
   const hasOutput = (obj) => {
     return obj !== undefined && obj.length > 0;
   };
 
   const [selectedTab, setSelectedTab] = useState(0);
-  const tabPanelOptions = {
-    height: contentHeight,
-    overflow: 'auto',
-    fontFamily: 'monospace, monospace',
-  };
-  const errorTabPanelOptions = { ...tabPanelOptions, color: 'red.500' };
-  const empty = { color: 'gray.200' };
-
-  const noStdOut = 'No output logs to display.';
-  const noStdErr = 'No error logs to display.';
-  const noErr = 'No errors to display.';
+  const emptyElement = (
+    <Box sx={{ p: 2, backgroundColor: 'gray.100' }}>Nothing to see here!</Box>
+  );
 
   return (
     <Box sx={{ mb: 4 }} pb={1}>
```

Comment:
In the future, I feel we should move all 'external' spacing control to caller (container) of the component rather than within the component to maximize the component's reusability

Reply:
Yep, I agree. I've been trying to move these whenever I modify the code.